js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -361,7 +361,7 @@ export class MiniAppBridge {
     return new Promise<string | null>((resolve, reject) => {
       return this.executor.exec(
         'sendMessageToContact',
-        { messageToContact: message },
+        { messageToContact: messageTrimmingBannerText(message) },
```

Comment:
I think it's a little strange to pass the whole message to this function when it only modifies one property. Maybe you could do something like this?
```suggestion
        { 
            messageToContact: {
                ...message,
                bannerMessage: trimBannerText(message.bannerMessage)
            }
         },
```

Reply:
thanks for the suggestion @corycaywood !